test/integration/noderesourcetopology_test.go

```diff
@@ -58,6 +58,10 @@ const (
 	reasonFailedScheduling = "FailedScheduling"
 )
 
+const (
+	testPodName = "topology-aware-scheduler-pod"
```

Comment:
"make it easier for maintain and more readable" - but also adds a hidden (and in this case tiny) interdependency between test cases :)
this is in general, in this case this seems a good change, however.